docs/user/customization.md

```diff
@@ -0,0 +1,122 @@
+      size: 500
+      type: io1
+    zones:
+    - us-west-2c
+
+networking:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  machineCIDR: 10.0.0.0/16
+  serviceCIDR: 172.30.0.0/16
+  type: OpenshiftSDN
+
+platform:
+  aws:
+    region: us-west-2
+    userTags:
+      costCenter: 7536
+      adminContact: jdoe
+```
+
+#### AWS Platform Customization
+
+The following options are available when using AWS:
+
+- `machines.platform.iamRoleName` - the IAM role that will be assigned to the machines of this pool
+- `machines.platform.rootVolume.iops` - the reserved IOPS of the root volume
+- `machines.platform.rootVolume.size` - the size (in gigabytes) of the root volume
+- `machines.platform.rootVolume.type` - the storage type of the root volume
+- `machines.platform.type` - the EC2 instance type
+- `machines.platform.zones` - a list of the availability zones that the installer will use when creating machines of this pool
+- `platform.aws.region` - the AWS region that the installer will use when creating resources
+- `platform.aws.userTags` - a map of keys and values that the installer will add as tags to all resources it creates
+
+### Kubernetes Customization (unmanaged)
+
+In addition to customizing OpenShift and aspects of the underlying platform, the installer allows arbitrary modification to the Kubernetes objects that are injected into the cluster. Note that there is currently no validation on the modifications that are made, so it is possible that the changes will result in a non-functioning cluster. The Kubernetes manifests can be viewed and modified using the `manifests` and `manifest-templates` targets.
+
+The `manifests` target will render the manifest templates and output the result into the asset directory. Perhaps the most common use for this target is to include additional manifests in the initial installation. These manifests could be added after the installation as a "Day 2" operation, but there may be cases where they are necessary beforehand.
+
+The `manifest-templates` target will output the pre-rendered manifest templates into the asset directory. This allows modification to the templates before they have been rendered, which may be useful to users who wish to reuse the templates between cluster deployments.
+
+### OS Customization (unmanaged)
+
+In rare circumstances, certain modifications to the bootstrap and other machines may be necessary. The installer provides the "ignition-configs" target, which allows arbitrary modification to the [Ignition Configs][ignition] used to boot these machines. Note that there is currently no validation on the modifications that are made, so it is possible that the changes will result in a non-functioning cluster.
+
+An example `worker.ign` is shown below. It has been modified to increase the HTTP timeouts used when fetching the generated worker config from the cluster. This isn't likely to be useful, but it does demonstrate what is possible.
+
+```json ignition
```

Comment:
Is this a thing?  I'd have expected:

    ```json

without `ignition`.  Checking [GitHub's rendered view][1], the JSON doesn't look syntax-highlighted to me.

[1]: https://github.com/openshift/installer/blob/9b4a9939913e0d106d27020b8bd8036f2bdbe378/docs/user/customization.md#os-customization-unmanaged

Reply:
From [the spec](https://spec.commonmark.org/0.28/#info-string):

> The first word of the info string is typically used to specify the language of the code sample, and rendered in the class attribute of the code tag. However, this spec does not mandate any particular treatment of the info string.

We do this in the Ignition docs so that GitHub renders them as JSON, but our site renders them as Ignition. I see the syntax highlighting in that link you provided. It's just not very colorful.